Error in make when test riscof example

ERROR | make: *** [/home/cat/riscv-arch-test/riscof-plugins/rv32/riscof_work/Makefile.Reference-sail_c_simulator:5: TARGET0] Error 1

which makes no Reference-sail_c_simulator.signature file.

> Can you attach the full output of running RISCOF?

Ok,sir.

```
cat@L:~/riscv-arch-test/riscof-plugins/rv32$ cd
cat@L:~$ source riscv_env/bin/activate
(riscv_env) cat@L:~$ cd riscv-arch-test/riscof-plugins/rv32
(riscv_env) cat@L:~/riscv-arch-test/riscof-plugins/rv32$ riscof run --config config.ini --suite ../../riscv-test-suite/ --env ../../riscv-test-suite/env

    INFO | ****** RISCOF: RISC-V Architectural Test Framework 1.25.3 *******
    INFO | using riscv_isac version : 0.18.0
    INFO | using riscv_config version : 3.20.0
    INFO | Reading configuration from: /home/cat/riscv-arch-test/riscof-plugins/rv32/config.ini
    INFO | Preparing Models
    INFO | Checked ISA file already exists
    INFO | Input-Platform file
    INFO | Loading input file: /home/cat/riscv-arch-test/riscof-plugins/rv32/spike_simple/spike_simple_platform.yaml
    INFO | Load Schema /home/cat/riscv_env/lib/python3.12/site-packages/riscv_config/schemas/schema_platform.yaml
    INFO | Initiating Validation
    INFO | No Syntax errors in Input Platform Yaml. :)
    INFO | Dumping out Normalized Checked YAML: /home/cat/riscv-arch-test/riscof-plugins/rv32/riscof_work/spike_simple_platform_checked.yaml
    INFO | Generating database for suite: /home/cat/riscv-arch-test/riscv-test-suite
    INFO | Database File Generated: /home/cat/riscv-arch-test/riscof-plugins/rv32/riscof_work/database.yaml
    INFO | Env path set to/home/cat/riscv-arch-test/riscv-test-suite/env
    INFO | Running Build for DUT
    INFO | Running Build for Reference

   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zcmop/src/c.mop.1-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zcmop/src/c.mop.11-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zcmop/src/c.mop.13-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zcmop/src/c.mop.15-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zcmop/src/c.mop.3-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zcmop/src/c.mop.5-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zcmop/src/c.mop.7-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zcmop/src/c.mop.9-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.0-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.1-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.10-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.11-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.12-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.13-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.14-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.15-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.16-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.17-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.18-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.19-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.2-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.20-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.21-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.22-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.23-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.24-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.25-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.26-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.27-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.28-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.29-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.3-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.30-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.31-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.4-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.5-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.6-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.7-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.8-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.r.9-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.rr.0-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.rr.1-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.rr.2-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.rr.3-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.rr.4-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.rr.5-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.rr.6-01.S
Test Selected without the relevant extensions being available on DUT.
   ERROR | Error in test: /home/cat/riscv-arch-test/riscv-test-suite/rv64i_m/Zimop/src/mop.rr.7-01.S
Test Selected without the relevant extensions being available on DUT.

    INFO | Selecting Tests.
    INFO | Running Tests on DUT.
    INFO | Running Tests on Reference Model.
   ERROR | make: *** [/home/cat/riscv-arch-test/riscof-plugins/rv32/riscof_work/Makefile.Reference-sail_c_simulator:5: TARGET0] Error 1
    INFO | Initiating signature checking.
Traceback (most recent call last):
  File "/home/cat/riscv_env/bin/riscof", line 7, in <module>
    sys.exit(cli())
             ^^^^^
  File "/home/cat/riscv_env/lib/python3.12/site-packages/click/core.py", line 1442, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cat/riscv_env/lib/python3.12/site-packages/click/core.py", line 1363, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/home/cat/riscv_env/lib/python3.12/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cat/riscv_env/lib/python3.12/site-packages/click/core.py", line 1226, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cat/riscv_env/lib/python3.12/site-packages/click/core.py", line 794, in invoke
    return callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cat/riscv_env/lib/python3.12/site-packages/click/decorators.py", line 34, in new_func
    return f(get_current_context(), *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cat/riscv_env/lib/python3.12/site-packages/riscof/cli.py", line 337, in run
    report_objects['results'] = framework.run(dut, base, ctx.obj.isa_file,
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cat/riscv_env/lib/python3.12/site-packages/riscof/framework/main.py", line 209, in run
    results = test.run_tests(dut, base, ispec['hart0'], pspec, work_dir, cntr_args)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cat/riscv_env/lib/python3.12/site-packages/riscof/framework/test.py", line 458, in run_tests
    result, diff = compare_signature(res, ref)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cat/riscv_env/lib/python3.12/site-packages/riscof/framework/test.py", line 46, in compare_signature
    file2_lines = open(file2, "r").readlines()
                  ^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/cat/riscv-arch-test/riscof-plugins/rv32/riscof_work/rv32i_m/A/src/amoadd.w-01.S/ref/Reference-sail_c_simulator.signature'
```

--- a/riscof-plugins/rv32/sail_cSim/riscof_sail_cSim.py
+++ b/riscof-plugins/rv32/sail_cSim/riscof_sail_cSim.py
@@ -124,6 +124,7 @@ class sail_cSim(pluginTemplate):
         # Enabling extensions that are disabled by default
         sail_config["extensions"]["Sv32"]["supported"] = True
         sail_config["extensions"]["Zcf"]["supported"] = True
+        sail_config["extensions"]["Svrsw60t59b"]["supported"] = False
 
         #For User-configuration: Replace this variable with your configuration. "/home/riscv-arch-test/custom_sail_config.json"
         sail_config_path = os.path.join(self.pluginpath, 'env', 'sail_config.json')

--- a/riscof-plugins/rv64/sail_cSim/riscof_sail_cSim.py
+++ b/riscof-plugins/rv64/sail_cSim/riscof_sail_cSim.py
@@ -118,7 +118,6 @@ class sail_cSim(pluginTemplate):
             exit(1)
 
         sail_config["base"]["xlen"] = int(self.xlen)
-        sail_config["base"]["mtval_has_illegal_instruction_bits"] = True
         sail_config["memory"]["pmp"]["grain"] = pmp_flags["pmp-grain"]
         sail_config["memory"]["pmp"]["count"] = pmp_flags["pmp-count"]
 

